core/src/migrations/000055-errorLevelForExports.ts

```diff
@@ -8,7 +8,6 @@ export default {
     await queryInterface.addColumn("exports", "errorLevel", {
       type: DataTypes.STRING(191),
       allowNull: true,
-      defaultValue: null,
```

Comment:
This change here is to remove the default(null) from the migration.  This isn't needed because a default of null is the default behavior in postgres.

Reply:
Makes sense.  I see that many of the other models got the `@AllowNull(true)` and `@DefaultValue(null)` decorators on columns (specifically on the `locked` column), but I don't see that on the `errorLevel` column in `exports`.  Just clarifying, your comment is saying that exports.errorLevel doesn't have those decorators on purpose, correct?